pydicer/utils.py

```diff
@@ -15,3 +20,68 @@ def hash_uid(uid, truncate=6):
     hash_sha = hashlib.sha256()
     hash_sha.update(uid.encode("UTF-8"))
     return hash_sha.hexdigest()[:truncate]
+
+
+def determine_dcm_datetime(ds):
+    """Get a date/time value from a DICOM dataset. Will attempt to pull from SeriesDate/SeriesTime
+    field first. Will fallback to StudyDate/StudyTime or InstanceCreationDate/InstanceCreationTime
+    if not available.
+
+    Args:
+        ds (pydicom.Dataset): DICOM dataset
+
+    Returns:
+        datetime: The date/time
+    """
+
+    if "SeriesDate" in ds and len(ds.SeriesDate) > 0:
```

Comment:
The 3 if blocks seem to be identical, maybe we could abstract it out to another function and pass in the 2 attributes (the time and date) for each if, to reduce redundancy?

Reply:
Agree, I'll rework this

Reply:
@dalmouiee  I've reworked this function and added a unit test :)